creativecloud/scripts/utils.js

```diff
@@ -59,21 +59,22 @@ function getDecorateAreaFn() {
   };
 
   async function loadLCPImage(area = document, { fragmentLink = null } = {}) {
-    const firstBlock = area.querySelector('body > main > div > div');
```

Comment:
@Ruchika4 In Milo also, the first div is getting selected :   `const firstDiv = document.querySelector('body > main > div:nth-child(1) > div');`

Does this change need to go in Milo?